<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addfieldwindow</class>
 <widget class="QWidget" name="addfieldwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Size:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_name">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>351</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_size">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>91</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Type: </string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_type">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>130</y>
        <width>72</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_isPK">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>170</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Is Primary key</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_addField">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>230</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Add field</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
